requirements/static/windows.in

```diff
@@ -23,3 +23,12 @@ setproctitle
 strict_rfc3339
 supervisor==3.3.5; python_version < '3'
 virtualenv
+watchdog
+
+# If running under windows, please uncomment the following 2 requirements before running
```

Comment:
You don't really need to do this anymore.
In fact, will `watchdog` be a windows packaging dependency?
If so, let's move it to `pkg/windows/reqs.txt` and recompile requirements.

Reply:
The comment at the end?